src/Components/Page/Page.js

```diff
@@ -36,8 +38,11 @@ export class Page extends Component {
         <Fragment>
           {showBreadcrumb && <Breadcrumb />}
           <Container>
+            {error &&
```

Comment:
notice that the errors have been moved to the top of the page where GDS says they should be